README.md

```
@@ -461,6 +482,13 @@ Distribuído sob licença ISC. Veja `LICENSE` para mais informações.
 
 > Este projeto é fornecido estritamente para fins educacionais e de pesquisa.
 
+### Hardening de Segurança Local
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)